### ISSUE TITLE
Shared hidedownload

### ISSUE THREAD

**Issue by marinofaggiana** [nextcloud/ios#729]
It's possible share a file/folder with the option "hide download", on client side is not possible the "open in".

**Comment by tobiasKaminsky** [comment]
As said in the other ticket, this is not the purpose of the hide download flag.
The flag only prevents people from downloading the file when they access the file via webbrowser with the share links.

--> this should do not change any behaviour on app side

**Comment by marinofaggiana** [comment]
yes, but this is a good moments for think a solution for client side ...
